src/browser/txml.js

```diff
@@ -0,0 +1 @@
+export { parse } from 'txml/dist/txml';
```

Comment:
It looks like this needs to be changed to:
```js
export { parse } from 'txml/txml';
```

Otherwise bundlers fail with something like this:
```
ERROR in ../node_modules/geotiff/src/browser/txml.js 1:0-39
Module not found: Error: Package path ./dist/txml is not exported from package /path/to/project/node_modules/txml (see exports field in /path/to/project/node_modules/txml/package.json)
```